doc/nrf/ug_nrf70_developing_powersave.rst

```diff
@@ -0,0 +1,323 @@
+A device in the PS mode starts the Service Period (SP) by sending a trigger frame that is a QoS Data/QoS Null frame.
+
+When the device receives a beacon with its AID in the TIM element, it initiates the SP by transmitting the trigger frame to the AP.
+The AP sends one or more individually addressed buffered frames to the device.
+The device remains active until the AP ends the SP by setting the End Of Service Period (EOSP) bit to ``1`` in the **QoS Control** field of the last QoS Data frame sent to the device.
+
+The following figure illustrates the unicast frame data retrieval mechanism in WMM Power Save mode:
+
+.. figure:: ../nrf/images/nRF70_ug_wmm_power_save.svg
+   :alt: WMM Power Save mode
+
+   WMM Power Save mode
+
+Legacy Power Save mode is the default option in nRF70 Series devices.
+The user or application can configure the WMM mode using the ``NET_REQUEST_WIFI_PS_MODE`` network management API.
+The WMM mode does not deliver a significant performance or power difference when compared to Legacy mode in Dynamic Power Save operation.
+
+The average power consumption of the device is affected by the DTIM period.
+The typical value is three beacons, for example, 307 ms for a beacon period of 100-time units.
+The higher DTIM period results in increased power saving and higher latency to the application data.
+
+Target Wake Time (TWT)
+**********************
+
+Target Wake Time is a new feature in Wi-Fi 6.
+It allows an access point and devices to wake up at the negotiated times.
+The access point and devices reach a TWT agreement that defines when a station is active and ready to receive and transmit data.
+
+TWT mechanism allows each device to negotiate its period with the AP to transmit and receive data packets.
+Stations will be active only at TWT Service Period and remain in the sleep mode for the rest of the time.
+
+.. note::
+   The device is not expected to be active for the DTIM beacon that do not receive any buffered broadcast or multicast frames.
+   Applications are expected to keep note of this and set up TWT sessions based on their traffic profile.
+
+An access point has more control over the network in TWT mode and decides which device is going to transmit and when.
+The AP decides how many and which Resource Units (RU) are to be used (an RU is a contiguous set of subcarriers).
+TWT offers more efficient scheduling of transmissions.
+
+Devices can remain in the sleep mode even longer, if required.
+Before Wi-Fi 6, device would sleep for DTIM period and then awake, exchange data, and go back to the sleep mode for DTIM period and continues the same.
+With TWT, a device can sleep for seconds, minutes, or even hours.
+Some devices can be configured to communicate once a day to perform a transmission and sleep for the rest of the day.
+
+The following figure illustrates the TWT wakeup sequence:
+
+.. figure:: ../nrf/images/nRF70_ug_twt.svg
+   :alt: TWT wakeup sequence
+
+   TWT wakeup sequence
+
```

Comment:
@udaynordic it is not clear what point you are trying to convey with this diagram. TWT1 and TWT2 are not referenced anywhere. Maybe a better diagram would appear later under the discussion of wake duration and wake interval.

Reply:
Will remove this and add diagram in twt key parameters section

Reply:
Let's keep it, but include additional text to describe it better.